3-UsingAPIs/Week1/assignment/ex4-pokerDiceAll.js

```diff
@@ -27,9 +27,10 @@ exercise file.
 const rollDie = require('../../helpers/pokerDiceRoller');
 
 function rollDice() {
-  // TODO Refactor this function
   const dice = [1, 2, 3, 4, 5];
-  return rollDie(1);
+  const arrayOfPromises = dice.map((die) => rollDie(die));
```

Comment:
This works fine but you can even make it more concise. The `.map` method calls its callback parameter function with the current element as the first argument. This is exactly what the `rollDie()` functions expects. So you can cut the arrow function as middle man.

```suggestion
  const arrayOfPromises = dice.map(rollDie);
```